### PR TITLE
Actions functionality commit

### PR DESCRIPTION
**Pull request by y-tst**

### CODE REVIEW

File: src/test/java/pages/jQueryUIPages/SortablePlaceholderPage.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import org.openqa.selenium.support.FindBy;
+import pages.BasePage;
+
+import java.util.List;
+import java.util.logging.Logger;
+
+public class SortablePlaceholderPage extends BasePage {
+
+    public static final String SORTABLE_PLACEHOLDER_URL = "https://jqueryui.com/sortable/#placeholder";
+    protected Logger logger = Logger.getGlobal();
```
Comment (olelysieev-clgx) [inline src/test/java/pages/jQueryUIPages/SortablePlaceholderPage.java]: remove this line, you use logger from Base page

File: src/test/java/tests/jQueryUITests/JQueryUIPageURLRemoteWebDrivertest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+public class JQueryUIPageURLRemoteWebDrivertest extends BaseTest {
+
+    @Test
+    public void jQueryUIPageURLForDraggableElementCheck() throws MalformedURLException {
+
+        DesiredCapabilities desiredCapabilities = new DesiredCapabilities();
+        desiredCapabilities.setPlatform(Platform.ANY);
+        desiredCapabilities.setBrowserName("chrome");
+
```
Comment (olelysieev-clgx) [inline src/test/java/tests/jQueryUITests/JQueryUIPageURLRemoteWebDrivertest.java]: remove commented code

File: src/test/java/pages/jQueryUIPages/SortablePlaceholderPage.java (hunk 1 above)
Comment (y-tst) [reply src/test/java/pages/jQueryUIPages/SortablePlaceholderPage.java]: I'm using the logger from BeaseTest in my tests, there is no logger in BasePage. That's why I decided to initiate it separately on the needed pages. Actually I can do this on BasePage and use it on pages

File: src/test/java/tests/jQueryUITests/JQueryUIPageURLRemoteWebDrivertest.java (hunk 2 above)
Comment (y-tst) [reply src/test/java/tests/jQueryUITests/JQueryUIPageURLRemoteWebDrivertest.java]: done

File: src/test/java/pages/jQueryUIPages/SelectableSerializePage.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    protected Logger logger = Logger.getGlobal();
+
+    @FindBy(xpath = "//iframe[@class='demo-frame']")
+    private WebElement frameElement;
+
+    @FindBy(xpath = "//span[@id = 'select-result']")
+    private WebElement selectableResumeElement;
+
+    @FindAll({@FindBy(xpath = "//li[@class='ui-widget-content ui-selectee']")})
+    List<WebElement> selectableList;
```
Comment (olelysieev-clgx) [inline src/test/java/pages/jQueryUIPages/SelectableSerializePage.java]: private

**Review by olelysieev-clgx — APPROVED** [review]

**Pull request merged**